Runtime/Services/BacktraceSession.cs

```diff
@@ -0,0 +1,317 @@
+﻿using Backtrace.Unity.Common;
+using Backtrace.Unity.Interfaces;
+using Backtrace.Unity.Json;
+using Backtrace.Unity.Model;
+using Backtrace.Unity.Model.JsonData;
+using Backtrace.Unity.Model.Session;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Backtrace.Unity.Tests.Runtime")]
+namespace Backtrace.Unity.Services
+{
+    internal sealed class BacktraceSession : IBacktraceSession
+    {
+        /// <summary>
+        /// Default number of retries
+        /// </summary>
+        public const int DefaultNumberOfRetries = 3;
+
+        /// <summary>
+        /// Submission url
+        /// </summary>
+        public string SubmissionUrl { get; set; }
+
+        /// <summary>
+        /// Determine if http client should ignore ssl validation
+        /// </summary>
+        public bool IgnoreSslValidation
+        {
+            get
+            {
+                return RequestHandler.IgnoreSslValidation;
+            }
+            set
+            {
+                RequestHandler.IgnoreSslValidation = value;
+            }
+        }
+
+        private int _numberOfDroppedEvents = 0;

```

Comment:
You probably want `_numberOfDroppedRequests`, since this counts requests, not events.

Reply:
That's right. Renaming variable. Thanks!